extras/knowledge-base.rst

```diff
@@ -135,3 +150,23 @@ available in the Zammad ticket composer. That means you can use the same :doc:`k
    .. |gry| raw:: html
 
       <svg xmlns="http://www.w3.org/2000/svg" viewBox="30 30 40 40" width="25" height="25" style="fill: #adadad"><path d="M57,36.39c0-.55.32-.69.71-.3L61,39.3c.39.38.26.7-.29.7H58a1,1,0,0,1-1-1ZM37,63V37a3,3,0,0,1,3-3H53a1,1,0,0,1,1,1v5a3,3,0,0,0,3,3h5a1,1,0,0,1,1,1V63a3,3,0,0,1-3,3H40A3,3,0,0,1,37,63Z"/></svg>
+
+Using answers in ticket articles
+--------------------------------
+
+As soon as the knowledge base contains one or more answers, you can use these
+just like text modules. Instead of ``::`` just use ``??`` to open the search
+modal. The search is done full text on both, answer body and title in all
```

Comment:
no comma

Reply:
There are two commas - you mean both of them?

Reply:
Sorry I meant the second one - unless I misunderstood the sentence? The idea is that there is a full search that looks through both the answer body and the title, right? Because then I would skip the comma